packages/components/src/internal/components/user/UserDetailHeader.tsx

```diff
@@ -1,40 +1,45 @@
 import React, { FC, ReactNode, useMemo } from 'react';
 import { Map } from 'immutable';
 
-import { PageDetailHeader, User } from '../../..';
+import { Container, PageDetailHeader, User } from '../../..';
 
 import { getUserLastLogin, getUserPermissionsDisplay } from './actions';
 
-interface HeaderProps {
+interface Props {
+    container?: Partial<Container>;
     dateFormat: string;
     description?: string;
     renderButtons?: ReactNode;
+    showFolderTitle?: boolean;
     title: string;
     user: User;
     userProperties?: Map<string, any>;
 }
 
-export const UserDetailHeader: FC<HeaderProps> = props => {
-    const { user, userProperties, title, dateFormat, renderButtons, description } = props;
+export const UserDetailHeader: FC<Props> = props => {
+    const { container, dateFormat, description, renderButtons, showFolderTitle, title, user, userProperties } = props;
     const lastLogin = useMemo(() => getUserLastLogin(userProperties, dateFormat), [dateFormat, userProperties]);
     const userDescription = useMemo(() => {
         return description || getUserPermissionsDisplay(user).join(', ');
     }, [description, user]);
 
     return (
-        <PageDetailHeader
-            user={user}
-            iconUrl={user.avatar}
-            title={title}
-            description={userDescription}
-            leftColumns={10}
-        >
-            {lastLogin && <div className="detail__header--desc">Last Login: {lastLogin}</div>}
+        <PageDetailHeader user={user} iconUrl={user.avatar} title={title} description={userDescription} leftColumns={9}>
+            {showFolderTitle && !!container?.title && (
+                <div className="detail__header--desc">
+                    <i className="fa fa-folder-open" />
+                    &nbsp;{container.title}
+                </div>
+            )}
+            {lastLogin && <div className="detail__header--desc pull-right">Last Login: {lastLogin}</div>}
```

Comment:
This is not a difference introduced here, but I notice that Biologics leaves off the `Last login` info while LKSM and FM include it.  Any reason we don't want to be consistent here?  I have no strong opinion one way or the other really.

Reply:
I agree it was an inconsistency. Namely, didn't have an easy way for LKB to utilize the `UserProvider` which provides `userProperties` (as you know). Added `useUserProperties` hook and updated app home pages to utilize it. See fdc4b028.

Reply:
Nice. Thanks.